tests/ui/tox.ini

```diff
@@ -0,0 +1,11 @@
+# content of: tox.ini , put in same dir as setup.py
+[tox]
+envlist = ui-tests
+[testenv]
+basepython = python2.7
+install_command = pip install --no-deps {packages}
+[testenv:ui-tests]
+passenv = DISPLAY
+install_command = pip install {packages}
+deps = -rrequirements.txt
+commands = py.test --verify-base-url --driver=Firefox test_discopane.py {posargs}
```

Comment:
`--driver=Firefox` should be picked up by `tests/ui/setup.cfg` so we can remove it from here. We could also add `--verify-base-url` to `setup.cfg` to avoid needing it here.